From 67ec58f2bfcc196d56ad5b4d553f541f92e1fc51 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Thu, 27 Jul 2017 08:23:50 +0100
Subject: [PATCH 1/2] Don't set DEFAULT_BASEDIR

This makes ioquake3 set it at runtime from argv[0], making it relocatable,
which is useful when building it into a Flatpak app.
---
 debian/rules | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 8025e584..1f2230e4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -37,7 +37,6 @@ OPTIONS := \
 	USE_VOIP=1 \
 	$(shell $(CURDIR)/debian/q3arch make ${DEB_HOST_GNU_CPU} ${DEB_HOST_GNU_SYSTEM}) \
 	BUILD_CLIENT_SMP=1 \
-	DEFAULT_BASEDIR=/usr/lib/ioquake3 \
 	BUILD_GAME_SO=1 \
 	BUILD_GAME_QVM=0 \
 	VERSION=$(DEB_VERSION)/$(DEB_VENDOR) \
-- 
2.17.1

